src/pages/Home/Content/ItemList/ItemCard/ItemCard.tsx

```diff
@@ -1,5 +1,5 @@
 import React from "react";
-import { FoodItem } from "../../../../../DummyData/DataTypes";
+import { FoodItem } from "../../../../../dummyData/dataTypes";
```

Comment:
you don't need the ../../ because i set the base path to src.
So you just need to import "dummyData/dataTypes"